.github/workflows/docker.yaml

```diff
@@ -0,0 +1,85 @@
+name: Build Docker Images
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+env:
+  DOCKER_HUB_SLUG: driveone/onedrive
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        flavor: [ fedora, debian, alpine ]
+        include:
+          - flavor: fedora
+            dockerfile: ./contrib/docker/Dockerfile
+            platforms: linux/amd64
+          - flavor: debian
+            dockerfile: ./contrib/docker/Dockerfile-debian
+            platforms: linux/amd64,linux/arm64,linux/arm/v7
+          - flavor: alpine
+            dockerfile: ./contrib/docker/Dockerfile-alpine
+            platforms: linux/amd64,linux/arm64
+
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Docker meta
+        id: docker_meta
+        uses: marcelcoding/ghaction-docker-meta@v1

```

Comment:
You could update this to v2. Yes for Ever release this will be tagged latest

Reply:
> You could update this to v2

will do